### ISSUE TITLE
support/analyzerHeadless script doesn't launch in headless mode and requires X11

### ISSUE THREAD

**Issue by MattMills** [NationalSecurityAgency/ghidra#4624]
**Describe the bug**
When using the analyzeHeadless script via SSH, the `-Djava.awt.headless=true` option is not added to the java command line, so AWT tries to start an X11 session and the "headless" analysis fails to start if X11 can't connect or if the X11 server goes away.

```
OpenJDK 64-Bit Server VM (build 11.0.16+8-post-Ubuntu-0ubuntu120.04, mixed mode)
Exception in thread "main" java.awt.AWTError: Can't connect to X11 window server using '[ip-removed]:0' as the value of the DISPLAY variable.
        at java.desktop/sun.awt.X11GraphicsEnvironment.initDisplay(Native Method)
        at java.desktop/sun.awt.X11GraphicsEnvironment$1.run(X11GraphicsEnvironment.java:102)
        at java.base/java.security.AccessController.doPrivileged(Native Method)
        at java.desktop/sun.awt.X11GraphicsEnvironment.<clinit>(X11GraphicsEnvironment.java:61)
        at java.base/java.lang.Class.forName0(Native Method)
        at java.base/java.lang.Class.forName(Class.java:315)
        at java.desktop/java.awt.GraphicsEnvironment$LocalGE.createGE(GraphicsEnvironment.java:101)
        at java.desktop/java.awt.GraphicsEnvironment$LocalGE.<clinit>(GraphicsEnvironment.java:83)
        at java.desktop/java.awt.GraphicsEnvironment.getLocalGraphicsEnvironment(GraphicsEnvironment.java:129)
        at java.desktop/sun.awt.X11.XToolkit.<clinit>(XToolkit.java:232)
        at java.base/java.lang.Class.forName0(Native Method)
        at java.base/java.lang.Class.forName(Class.java:315)
        at java.desktop/java.awt.Toolkit$2.run(Toolkit.java:588)
        at java.desktop/java.awt.Toolkit$2.run(Toolkit.java:583)
        at java.base/java.security.AccessController.doPrivileged(Native Method)
        at java.desktop/java.awt.Toolkit.getDefaultToolkit(Toolkit.java:582)
        at java.desktop/java.awt.Taskbar.isTaskbarSupported(Taskbar.java:241)
        at ghidra.Ghidra.main(Ghidra.java:44)
```

In my case I was running a script to do several analysis in a screen session, and then rebooted the machine where X11 was running, thus the analysis failed with unable to connect to X11.

**To Reproduce**
1. Set a DISPLAY= variable to an occasionally accessible X11 server
2. Try to run a headless analysis

**Expected behavior**
It would make a lot more sense to start java in headless mode, unless there is something I'm missing... I was able to work around this by adding `VMARG_LIST+="-Djava.awt.headless=true "` to the analyzerHeadless script config.

**Screenshots**
N/A

**Attachments**
N/A

**Environment (please complete the following information):**
```
root@hippo:~# cat /etc/lsb-release 
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=20.04
DISTRIB_CODENAME=focal
DISTRIB_DESCRIPTION="Ubuntu 20.04.4 LTS"
root@hippo:~# uname -a
Linux hippo 5.15.0-41-generic #44~20.04.1-Ubuntu SMP Fri Jun 24 13:27:29 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux
root@hippo:~# java --version
openjdk 11.0.16 2022-07-19
OpenJDK Runtime Environment (build 11.0.16+8-post-Ubuntu-0ubuntu120.04)
OpenJDK 64-Bit Server VM (build 11.0.16+8-post-Ubuntu-0ubuntu120.04, mixed mode, sharing)
root@hippo:~# ls -al /zpool0/share/ghidra_
ghidra_10.1.5_PUBLIC/              ghidra_10.1.5_PUBLIC_20220726.zip 
```
Ghidra zip is from github release.

**Additional context**
N/A

**Comment by ryanmkurtz** [comment]
This has been addressed in the upcoming Ghidra 10.2 release.  We still prefer not to set that JVM argument and would rather rely on Java to tell us the true display state of the machine.  The code in `Ghidra.java` has been improved so it doesn't blow up on machine configurations like yours.

**Comment by ryanmkurtz** [comment]
https://github.com/NationalSecurityAgency/ghidra/blob/dc76aa811ed6831307f4690f61753b802f922786/Ghidra/Framework/Utility/src/main/java/ghidra/Ghidra.java#L41-L51
